src/Components/Components/src/EventCallbackFactoryBinderExtensions.cs

```diff
@@ -1379,10 +1379,10 @@ public static class EventCallbackFactoryBinderExtensions
         return factory.Create<ChangeEventArgs>(receiver, callback);
     }
 
-    private static EventCallback<ChangeEventArgs> CreateBinderCore<T>(
+    private static EventCallback<ChangeEventArgs> CreateBinderCoreAsync<T>(
```

Comment:
Why the `Async` suffix?

Reply:
The compiler can't decide the right overload otherwise.